make.sh

```diff
@@ -9,4 +9,9 @@ fi
 PATH="$(cd -- "$(dirname "$0")" && pwd)/ci/sub/bin:$PATH"
 cd -- "$(dirname "$0")"
 
+if ! $(go version | grep -qF "1.18."); then
+  printferr "You need go 1.18 to build d2.\n"
```

Comment:
```suggestion
  echoerr "You need go 1.18 to build d2."
```
Just realized you don't need printf anymore.